### DISCUSSION TITLE
GTK Theme being force applied on buttons

### DISCUSSION THREAD

**Discussion by derpyzza** [elkowar/eww#703] [Q&A]
Hello, im working on a config and all the buttons in my config are getting this strange white border on them. I am not manually setting the white borders in my config anywhere, and if i use the eww inspector and activate the "Dark variant"  option in the "Visuals" tab, the white border goes away. The thing is, however, that i need to manually activate the 'Dark variant' option every time the widget window closes and opens back up again, which is super tedious. Is there any way to stop the default gtk themes from interfering with my widgets? Thank you :)

(This is what it looks like)
![20230306_17h02m54s_grim](https://user-images.githubusercontent.com/68068652/223106279-e8c682ba-547d-41a1-b2b2-9ed3d4e09102.png)

(And this is with the 'Dark Variant' setting on)
![20230306_17h09m15s_grim](https://user-images.githubusercontent.com/68068652/223106487-81fb712e-e30b-4edb-835d-9955b4b6b9b5.png)

**Comment by viandoxdev** [comment]
Consider using

```css
* {
    all: unset;
}
```

to disable the default gtk theme styling when using your own css.
